src/frontend/src/components/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import { Link } from "react-router-dom";
```

Comment:
Auf welchen Pages wird jetzt der footer Verwendet? Ich finde alles was auf allen Seiten verwendet werden kann, gehört in den elements ordner, und dann gibt es pro Page Ordner noch einen "components" Ordner für Komponenten, die zu nur der Page gehören.

Reply:
Diese Struktur verwenden grob bei uns im Team, und es macht das ganze meiner Meinung nach etwas übersichtlicher